README.md

```diff
@@ -2,12 +2,77 @@
 
+- [Project summary](#project-summary)
+- [Updates](#updates)
+- [Getting started](#getting-started)
+  - [Downloading the project](#downloading-the-project)
+  - [Setting up an emulator](#setting-up-an-emulator)
+- [Uploading to the Play Store](#uploading-to-the-play-store)
+  - [Part 1: Generating a Signed APK](#generating-a-signed-apk)
+  - [Part 2: Creating a Release in the Play Console](#part-2-creating-a-release-in-the-play-console)
+  - [Part 3: Filling in app details on App Store Connect](#part-3-filling-in-app-details-on-app-store-connect)
+
 ## Project summary
 
 This project is the mirror version of the iOS Science Student Success Centre app, for Android! It allows students to view all of the events that are shown on the SSSC's website and quickly access the SSSC's resources page. 
 Event data for both the Android and iOS apps are retrieved using a GET request to the SSSC Server (linked above) using the `/events` endpoint, to ensure their data is sychronized. 
 
+## Updates
+
+(Refer to the ios app for the following features)
+1. Add ability to back up grades
+2. Add colors to a course
+3. Add search bars to the top app bar(Terms screen, Calculator screen)
+4. Find a more user friendly/appealing way to delete list items. Current implemention is long press on the item.
+
+## Getting started
+
+There are a few different steps you'll need to go through to work on the app. The first step should be to have Android Studio installed on your computer if you dont have it already. You can follow the download instrauction [here](https://developer.android.com/studio/install).
+
+### Downloading the project
+
+The first thing you'll need to do in order to work on the app is to download it! Here are the steps to get it onto your machine properly.
+
+1. Clone the repository: `git clone https://github.com/ScienceStudentSuccessCentre/SSSC-Android-app.git`
+2. Open the project in Android Studio.
+
+### Setting up an emulator
+If you plan to use an emulator follow this [tutorial](https://developer.android.com/studio/run/emulator) that walk you through setting up an emulator and actually running the app!
+
+### Uploading to the Play Store
+
+When you are satisfied with the new features you have added it's time to release your update to the Play Store. Here is a quick sumnary of the steps you'll need to take in order to upload a new build to the Play Store and release it. The official guide can be found [here](https://developer.android.com/studio/publish)
+
+## Part 1: Generating a Signed APK
+
+Google requires that you app is signed with a certain key. In this section we'll go through getting a signed APK from Android Studio.
+
+* Note make sure you have the keystore file(.jks) before you start this process *
+
+1. First step is to increment the versionCode and versionName attributes in the app level gradle file.
```

Comment:
Can you explicitly put the filename here? Otherwise they might get the Gradle files confused